warehouse/internal/repo/upload.go

```diff
@@ -137,9 +139,8 @@ func (uploads *Uploads) CreateWithStagingFiles(ctx context.Context, upload model
 		}
 	}()
 
-	row := tx.QueryRowContext(
-		ctx,
-
+	var uploadID int64
+	err = tx.QueryRow(
```

Comment:
Why we have removed ctx from here ?

Reply:
because it's the same context that's used to create the transaction.